openwisp_firmware_upgrader/tests/test_admin.py

```diff
@@ -109,7 +109,7 @@ def test_upgrade_selected_error(self):
     def test_upgrade_intermediate_page_related(self):
         self._login()
         env = self._create_upgrade_env()
-        with self.assertNumQueries(17):
+        with self.assertNumQueries(15):
```

Comment:
The number of queries changes because we removed `VersionAdmin` from `BuildAdmin` class